verifier/src/report_body.rs

```diff
@@ -362,4 +397,27 @@ mod test {
 
         assert_eq!(verifier.verify(&isv_svn).is_some().unwrap_u8(), 1);
     }
+
+    #[test]
+    fn miscellaneous_select_success() {
+        let miscellaneous_select = MiscellaneousSelect::from(REPORT_BODY_SRC.misc_select);
+        let verifier = MiscellaneousSelectVerifier::new(miscellaneous_select.clone());
```

Comment:
MiscellaneousSelect is a u32, created https://github.com/mobilecoinfoundation/sgx/issues/283 to derive copy until then using `clone()`